test/videojs-http-streaming.test.js

```diff
@@ -3687,27 +3687,29 @@ QUnit.test('cleans up the buffer when loading VOD segments', function(assert) {
     // tick.
     this.clock.tick(2);
 
+    assert.ok(this.requests[3].aborted, 'request aborted during seek');
+
     // request second segment, and give enough time for the source buffer to process removes
     return requestAndAppendSegment({
-      request: this.requests[3],
+      request: this.requests[4],
       mediaSource: mpc.mediaSource,
       segmentLoader: mpc.mainSegmentLoader_,
       clock: this.clock
     });
   }).then(() => {
-    assert.equal(audioRemoves.length, 1, 'one audio remove');
-    assert.equal(videoRemoves.length, 1, 'one video remove');
+    assert.ok(audioRemoves.length, 'audio removes');
+    assert.ok(videoRemoves.length, 'video removes');
     // segment-loader removes at currentTime - 30
-    assert.deepEqual(
-      audioRemoves[0],
+    assert.deepEqual(audioRemoves, [
+      { start: 0, end: 40 },
```

Comment:
Yes exactly. The first remove comes from the `setCurrentTime` call, caused by `player.currentTime(120)`. In the original test this one was ignored because we did not mock `sourceBuffer.remove` early enough. The second remove comes from `trimBackBuffer_` which I assume is what this test was originally testing. The final remove comes from https://github.com/videojs/http-streaming/blob/225d1278a48f6be600c08638380219a2915c578c/src/segment-loader.js#L2569 as the "guess" after our `requestAndAppendSegment` is clearly off after the append.

Reply:
Might be worth adding (at least the second one) as a comment, in case anyone else stumbles into it (or me again). But otherwise things look good.

Reply:
A quick comment here makes sense.